sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/utils.py

```diff
@@ -102,9 +118,9 @@ def normalized_data_body(data, **kwargs):
     return [encode_str(data, encoding)]
 
 
-def normalized_sequence_body(sequence):
+def normalized_sequence_body(sequence): # pylint:disable=inconsistent-return-statements
     # A helper method to normalize input into AMQP Sequence Body format
-    if isinstance(sequence, list) and all([isinstance(b, list) for b in sequence]):
+    if isinstance(sequence, list) and all([isinstance(b, list) for b in sequence]): # pylint:disable=use-a-generator
```

Comment:
I noticed that you used all((...)) in other places instead of all([...]). Is there a reason you used all(list()) here instead of all(tuple())?

Reply:
Ah because this was before I realized using tuple() would resolve this, let me fix this one